### ISSUE TITLE
NUnit1001 - false positive for arguments with nullable enum

### ISSUE THREAD

**Issue by mischube** [nunit/nunit.analyzers#941]
When i try to use the TestCase attribute with a nullable enum as parameter i get NUnit1001 altough the type matches

package version 4.11

```csharp
    public enum MyEnum
    {
        Case1
    }

    [TestCase(MyEnum.Case1)]
    [TestCase(null)]
    public void FooTest(MyEnum? param2)
    {
        
    }
```
